config/locales/overrides/decidim.en.yml

```diff
@@ -1,8 +1,8 @@
 en:
   decidim:
     menu:
-      processes: 'Participate'
-      more_information: 'About'
+      processes: Participate
```

Comment:
The single quotes are optional, so I removed them where not required (i.e. when a string doesn't have a `:`). This means we don't have escape single quotes when they occur naturally within a string.